exporter/elasticsearchexporter/internal/objmodel/objmodel.go

```diff
@@ -175,6 +176,15 @@ func (doc *Document) AddAttribute(key string, attribute pcommon.Value) {
 	}
 }
 
+// AddEvents converts and adds span events to the document.
+func (doc *Document) AddEvents(key string, events ptrace.SpanEventSlice) {
+	for i := 0; i < events.Len(); i++ {
+		e := events.At(i)
+		doc.Add(flattenKey(key, e.Name()+".time"), TimestampValue(e.Timestamp().AsTime()))
+		doc.AddAttributes(flattenKey(key, e.Name()), e.Attributes())
```

Comment:
Do you care about `dropped_attributes_count`?

Reply:
I don't care `dropped_attributes_count ` actually, But, I will add it here to make the event store more complete.

Reply:
I would recommend dropping  `dropped_attributes_count` as it's pretty misaligned with the recommendation for the OTLP-Elastic schema translations in the spec. It'll be less to break going forward